applications/nightly_build/test_keras_applications_v2.py

```diff
@@ -56,6 +56,12 @@ def test_InceptionV3(self):
         model = InceptionV3(include_top=True)
         res = run_image(model, self.model_files, img_path, target_size=299, tf_v2=True)
         self.assertTrue(*res)
+    
+    def test_InceptionResNetV2(self):
+        InceptionResNetV2 = keras.applications.inception_resnet_v2
```

Comment:
InceptionResNetV2 = keras.applications.InceptionResNetV2?

Reply:
It should be `keras.applications.inception_resnet_v2. InceptionResNetV2`, sorry about that.